### ISSUE TITLE
Does this work for North American users?

### ISSUE THREAD

**Issue by danielwelch** [robinostlund/volkswagencarnet#104]
I'm running into a bug during the login process, initially discovered while trying to setup the custom component for home assistant. I tried debugging by turning to the underlying library, but I'm getting the same error message when using this library. I've checked my credentials and am able to use VW CarNet via the web browser and iOS app without issue. Here's the output I get when running the example from the README (with my credentials replaced, of course):
```
➜ python3 test-vw.py
DEBUG:asyncio:Using selector: KqueueSelector
DEBUG:volkswagencarnet:Using service <https://www.portal.volkswagen-we.com/>
ERROR:volkswagencarnet:Failed to login to carnet, 'NoneType' object has no attribute 'get'
```
Am I missing something obvious?

**Comment by danielwelch** [comment]
I was able to trace the error back to a BeautifulSoup parsing error during login, which was being thrown essentially because I didn't have an account set up in the way this library seems to expect. For background, I'm in the U.S. and have a Volkswagen Atlas with fully functioning car-net portal. Essentially, I had to make an account via we-connect (which didn't offer the U.S. as a location selection), although I already had a fully functioning account with Volkswagen car-net. Now the test script works without throwing errors, but shows that I have no cars. 

The library seems to work for cars accessible here: https://www.portal.volkswagen-we.com/portal/en_GB/web/guest/home
But not here: https://carnet.vw.com/

I guess this just doesn't work for North American users? Seems like a separate/distinct login flow.
